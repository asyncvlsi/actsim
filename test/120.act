pint N = 4;

defproc test (chan!(int<N>) C)
{
   chp {
      *[ C!0 ]
   }
}
