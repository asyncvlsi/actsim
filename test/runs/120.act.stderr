WARNING: test<>: substituting chp model (requested prs, not found)
Operation only works for a circuit object, not parameter `N'
Execution aborted.
Stack trace:
	called from: get
	called from: -top-level-
